scrapy_poet/utils.py

```diff
@@ -39,21 +39,21 @@ def parse(self, response: DummyResponse):
 
     If your ``PageObjectInputProvider`` doesn't need a request, you simply
     don't need to list it as a dependency. But if you need, for example, the
-    original request's URL, you can use ``DummyResponse`` instead of
+    original request's URL, you can use ``Request`` instead of
```

Comment:
I'm not sure a section about Request for PageObjectInputProvider makes sense in DummyResponse docstring; DummyResponse has nothing to do with providers now, it is for spider writers who need to skip downloads.